Cargo.toml

```diff
@@ -26,4 +26,11 @@ description = "A library for safe, in-place construction of Rust (and C++!) obje
 
 [features]
 alloc = []
+cxx_support = [ "cxx" ]
```

Comment:
Can we just not call the feature "cxx" and call it a day? AIUI each dependency implicitly names a feature for itself.

Reply:
TIL! Will do.